packages/gatsby-theme-docs/src/templates/release-note.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { graphql } from 'gatsby';
+
+const ReleaseNote = (props) => {
```

Comment:
All template pages should use the same "setup", meaning the different providers, etc.